roles/openshift_storage_glusterfs/tasks/glusterfs_deploy.yml

```diff
@@ -87,6 +87,7 @@
       IMAGE_VERSION: "{{ glusterfs_version }}"
       NODE_LABELS: "{{ glusterfs_nodeselector }}"
       CLUSTER_NAME: "{{ glusterfs_name }}"
+      GB_GLFS_LRU_COUNT: "{{ glusterfs_block_max_host_vol }}"
```

Comment:
Probably outside the scope of this PR, but we should see about changing this module to accept a dict of kv pairs.

Reply:
Definitely outside scope, but also agree. :) I was just following the pattern I've been seeing in other roles with multiple config options.